### PR TITLE
BUG: backport as_inexact kwarg to _asarray_validated

### PR DESCRIPTION
**Pull request by ev-br**
Fixes gh-5191

This PR is against the `maintanance/0.16.x` branch.

### CODE REVIEW

File: doc/sphinxext [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 447dd0b59c2fe91ca9643701036d3d04919ddc7e
+Subproject commit 7b17004d436e8440496aa47a2ccd9e2c4ca8d2ed
```
Comment (juliantaylor) [inline doc/sphinxext]: looks like an accidental commit, should be removed

Comment (juliantaylor) [inline doc/sphinxext]: or maybe in a separate PR

Comment (ev-br) [inline doc/sphinxext]: Done.
[note to self: git commit -am and git submodules, darn]

**Comment by ev-br** [comment]
Test error is a timeout

**Comment by rgommers** [comment]
Merged, thanks Evgeni.

**Pull request merged**